src/process.rs

```diff
@@ -50,11 +86,20 @@
 ///
 /// let mut parser = Rdp::new(StringInput::new("b"));
 ///
+/// // Notice that you have to call b() here so that there are some tokens in the queue
 /// parser.b();
 /// parser.ab();
 /// # }
 /// ```
 ///
+/// The reason for using a `panic!` instead of a `Result` is because this error is not common
```

Comment:
I guess you could say here that the `panic!` useful for debugging purposes and I would definitely mention the fact that match mistakes should not be handled at runtime here (with `Result`) because this is what the parser should be doing.